src/datatypes/src/value.rs

```diff
@@ -77,6 +77,8 @@ pub enum Value {
     Interval(Interval),
 
     List(ListValue),
+
+    Json(Bytes),
```

Comment:
I think we can remove the type here. We can considered the JSON as kind of logical type.